index.html

```diff
@@ -534,15 +538,42 @@ <h5 class="modal-title" id="exampleModalLabel">Granola with cherries and strawbe
 			</div>
      	</section>
     	     
-    	<section class="dashboard" style="display: none;">
-    		<div>
-    			<ul id="show-cart">
-    				
-    			</ul>
-    			<button id="clear-cart">Clear Cart</button>
-    			<div>Total Cart: $ <span id="total-cart"></span></div>
-    		</div>
-    	</section>
+    	<div class="modal fade modal-dashboard" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+		  <div class="modal-dialog modal-lg modal-dialog-dashboard">
+			<div class="modal-content modal-content-dashboard">
+				<div class="row">
+					<button type="button" class="close close-dashboard" data-dismiss="modal" aria-label="Close">
+									<span aria-hidden="true">&times;</span>
```

Comment:
I'm not sure if it's because of how github shows spaces, but the indentation seems broken here. Can you please check?

Reply:
The indentation for that <span> was indeed broken. But other than that, the indentation seems fine in my code. For example the table indentation has no problem when i open it on my local computer.